lib/ProductOpener/Products.pm

```diff
@@ -1001,6 +1001,25 @@ sub compute_sort_keys ($product_ref) {
 	return;
 }
 
+sub created_function($mongodb_to_mongodb, $product_ref, $products_collection, $data_root, $path) {
```

Comment:
Could you rename the function "update_existing_product_without_creating_a_new_revision()" instead of "created_function()", so that it's easier to understand what it does?